packages/core/src/ui/components/ImageWithFallback/ImageWithFallback.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+
+export interface ImageWithFallbackProps
+  extends React.DetailedHTMLProps<React.ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement> {
+  defaultSrc: string;
```

Comment:
Matching name of the component would make it more intuitive
```suggestion
  fallbackSrc: string;
```

Reply:
done here https://github.com/input-output-hk/lace/pull/1263/commits/bbc30a91464477195fc51e1564e9b11e6c0295ce